roles/pulibrary.nginxplus/files/conf/http/researchdata_prod.conf

```diff
@@ -1,39 +1,39 @@
 $ANSIBLE_VAULT;1.1;AES256
```

Comment:
I am assuming that the redirection here is temporary, until we have a dedicated app. for managing research data.